docs/sources/architecture/distributor.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Distributor
+weight: 1000
+---
+# Distributor Component
+
+This document builds upon the information in the [Loki Architecture](./) page.
+
+## Where does it live?
+
+The distributor is the first component on Loki's write path. It's responsible for validating, preprocessing, and applying a subset of rate limiting to incoming data before sending it to the ingester component. 
```

Comment:
first component downstream from any gateways providing auth or load balancing. The distributors require something upstream to do proper load balancing of incoming data